gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -342,6 +350,16 @@ def add_arguments(self, parser):
               'pass over all variants. Additionally, this flag will resolve '
               'conflicts for all headers as if `allow_incompatible_types` was '
               'true.'))
+    parser.add_argument(
+        '--run_with_garbage_collection',
+        type='bool', default=True, nargs='?', const=True,
+        help=('If set, in case of failure or cancellation, the VMs running '
+              'VEP annotation will be cleaned up automatically.'))
+    parser.add_argument(
+        '--number_of_variants_per_shard',
+        type=int, default=20000,
+        help=('The maximum number of variants written to each shard if '
```

Comment:
It is not clear why shard refers to here, therefore the description is not very helpful. Maybe explain how this number affect performance, and why someone should change it.

Reply:
Updated.